mk/spksrc.cross-env.mk

```diff
@@ -6,6 +6,7 @@ ENV += INSTALL_PREFIX=$(INSTALL_PREFIX)
 
 ifeq ($(strip $(REQUIRE_KERNEL)),1)
 ENV += REQUIRE_KERNEL_MODULE="$(REQUIRE_KERNEL_MODULE)"
+ENV += REQUIRE_KERNEL_HEADERS="$(REQUIRE_KERNEL_HEADERS)"
```

Comment:
I suppose this change was unintentially added from another branch...

Reply:
argh yes. sry. will remove it now.